docs/docs-content/release-notes.md

```diff
@@ -9,6 +9,60 @@ sidebar_custom_props:
 tags: ["release-notes"]
 ---
 
+## Feb 16, 2024 - Release 4.2.12
+
+### Bug Fix: IaaS Cluster Repaves Causing Cluster Downtime
+
+#### Affected services
+
+IaaS clusters in Palette 4.2.x prior to 4.2.12, including Palette SaaS, self-hosted Palette/VerteX, as well as dedicated
+instances. Affected cluster types include the following:
+
+- AWS IaaS (not EKS)
+- Azure IaaS (not AKS)
+- Google IaaS (not GKE)
+- MAAS
+- vSphere
+- OpenStack Managed Kubernetes clusters on EKS, GKE and AKS are not affected. Edge clusters are not affected.
+
+#### Issue Summary
+
+We identified an issue related to cluster repaves in Palette 4.2.x. During a cluster upgrade that required a repave, the
+Palette Agent running within the clusters would delete all the worker nodes within a worker pool before provisioning new
+worker nodes This results in the worker pool being down during an upgrade. All workloads within the pool will be offline
+during the upgrade.
+
+If the cluster is configured to enable updating worker pools in parallel, then this can result in all services on the
+cluster becoming unavailable.
+
+#### Customer Guidance
+
+This issue has been addressed in Palette 4.2.12 and its corresponding Palette Agent version 4.2.4. To identify whether
+your cluster is using an affected agent version:
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+2. From the left **Main Menu**., click **Clusters**. And then select your cluster to view cluster details.
+3. At the bottom of the cluster details page, you can see the Agent version used by your cluster. If your Agent version
+   is any of the following versions, your cluster is still susceptible to this issue: 4.2.0, 4.2.1, 4.2.2, 4.2.3
+
+::: warning
+
+Ensure that you do not initiate any cluster repaves as long as you are using an affected agent version. Changes in the
+OS or the Kubernetes layer would initiate an cluster repave attempt. When you get the cluster repave notification,
+reject the repave.
+
+:::
+
+**If you are not using an affected agent version**, no action is required on your part. If you plan to upgrade to 4.2.x
+in the future, ensure you upgrade to a version of Palette that's 4.2.12 or later.
+
+**If you are using an affected agent version**, first make sure that your Palette version is 4.2.12 or newer. Once you
+have confirmed your Palette version, unpause Agent upgrades for your cluster if they are paused. To learn how to toggle
```

Comment:
Would this not be the first step? To pause platform upgrades?

Reply:
So pausing platform upgrades when you have an affected agent version won't do you any good because the agent version has this repave issue. I think Customer Success have already advised some customers who are using agent 4.1.x to pause upgrade so they don't upgrade to 4.2.x, which has the issue. 

Once the fix goes live, they no longer need to pause the upgrade. Some might still choose to for other reasons, but if they are using an affected version, they need to unpause it.